smoothie-lifecycle-viewmodel-ktp/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+repositories {
+  google()
+  jcenter()
+}
+
+android {
+  compileSdkVersion 28
+
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_7
+    targetCompatibility JavaVersion.VERSION_1_7
+  }
+
+  defaultConfig {
+    minSdkVersion 15
+  }
+
+  sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
```

Comment:
we still have some kotlin code in src/main/java. I will merge the PR but I would like to move it to src/main/kotlin